llvm/include/llvm/Transforms/udachinaPass/udachinaPass.h

```diff
@@ -0,0 +1,21 @@
+#ifndef LLVM_TRANSFORMS_UDACHINAPASS_H
+#define LLVM_TRANSFORMS_UDACHINAPASS_H
+#define DEBUG_TYPE "udachinaPass"
```

Comment:
@anastasiaudachina, please move it into cpp to avoid name conflicts with other passes